### PR TITLE
Fix to use lower verbose name for exportable IDs

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: templates/orgs/org_export.haml [hunk 1]
```diff
@@ -42,11 +42,11 @@
               -for exportable in bucket
                 -ifchanged exportable|verbose_name_plural|capfirst
                   %h4.flex
-                    %input.toggle-all(type="checkbox" id="{{forloop.parentloop.counter0}}-{{ exportable|verbose_name_plural}}" style="margin-top: 8px !important;")
```
Comment (rowanseymour) [inline templates/orgs/org_export.haml:45]: hmm does this work when viewing site in a different language?

Comment (rowanseymour) [reply templates/orgs/org_export.haml:45]: I just checked and answer is no...

Comment (rowanseymour) [reply templates/orgs/org_export.haml:45]: @norkans7 we need a new filter which doesn't use localized strings, something like

```python
@register.filter
def class_name(obj):
    return obj.__class__.__name__
```

Comment (norkans7) [reply templates/orgs/org_export.haml:45]: I think we need a new meta variable that is it not localized

Comment (rowanseymour) [reply templates/orgs/org_export.haml:45]: Why? I don't think you can just define new meta variables. Why can't we use that filter and add `s` if necessary to get plural?

Comment (norkans7) [reply templates/orgs/org_export.haml:45]: Yes, I am  using a filter

File: temba/utils/templatetags/temba.py [hunk 2]
```diff
@@ -39,6 +39,20 @@
...
+    Flow: "flows",
+    Campaign: "campaigns",
+    CampaignEvent: "campaign_events",
+    Trigger: "triggers",
+    ContactGroup: "groups",
+}
+
+
+@register.filter
+def object_class_plural(o):
```
Comment (rowanseymour) [inline temba/utils/templatetags/temba.py:52]: I want to get this merged so it's fine but it seems odd that `object_class_plural` doesn't return `ContactGroups` etc.. doesn't really do what the name implies. I would have this return `o.__class__.__name` and then if you want it lowercase in a template use `|lower`

Comment (norkans7) [reply temba/utils/templatetags/temba.py]: Yeah, I just noticed that after pushing

Comment (norkans7) [reply temba/utils/templatetags/temba.py]: I called `object_class_name` as class name in HTML sounds more like CSS class

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**